config/initializers/devise.rb

```diff
@@ -9,7 +9,7 @@
     value: session_id,
     expires: 1.year.from_now,
     httponly: true,
-    secure: (Rails.env.production? || ENV['LOCAL_HTTPS'] == 'true'),
+    secure: ((Rails.env.production? || ENV['LOCAL_HTTPS'] == 'true') && !ENV['LOCAL_DOMAIN'].ends_with?('.onion')),
```

Comment:
With so much repetition, can this be extracted to another function or config value?